torch/_refs/__init__.py

```diff
@@ -1772,6 +1774,10 @@ def addcmul(
             utils.is_weakly_lesser_type(type(value), python_type),
             lambda: f"value argument of type {type(value)} cannot be safely cast to type {python_type}!",
         )
+    # torch.addcmul(torch.tensor(2), torch.tensor(3), torch.tensor(4), value=2.6)
+    #   == tensor(26)
+    # In core, this is allowed but might fail later if the value cannot be
+    # safely converted to int64_t without overflow.
```

Comment:
Hit this case while testing foreach_addcmul - in core, 2.6 is converted to 2. With refs, we raise an error. With other values, it might also fail in core. The ref is correct.

addcdiv is not affected because it raises an error due to integer div.